Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/WebView.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
+{
+	using FormsElement = Forms.WebView;
+
+	public enum MixedContentHandling
+	{
+		AlwaysAllow = 0,
+		NeverAllow = 1,
+		CompatibilityMode = 2
+	}
+
+	public static class WebView
+    {
+		public static readonly BindableProperty MixedContentModeProperty = BindableProperty.Create("MixedContentMode", typeof(MixedContentHandling), typeof(WebView), MixedContentHandling.NeverAllow);
```

Comment:
@alanag13 just a small thing here - any chance we can use nameof(MixedContentMode) here to avoid the string literal?